app/src/main/java/com/example/next_show/fragments/ShowDetailFragment.java

```diff
@@ -177,27 +184,13 @@ private void fetchShowPoster(ImageView iv) {
         client.get(getUrl, new JsonHttpResponseHandler() {
             @Override
             public void onSuccess(int i, Headers headers, JSON json) {
-                Log.d(TAG, "onSuccess");
-
-                // get json object
-                JSONObject jsonObj = json.jsonObject;
-
-                // get results and save to show object
-                try {
-                    String path = jsonObj.getString("backdrop_path");
-                    currShow.setImageUrl(path);
```

Comment:
Same here -- you could just process the image, right?  Or do you expect that others will want to parse a different image type?